geomesa-convert/geomesa-convert-xml/src/main/scala/org/locationtech/geomesa/convert/xml/XMLConverter.scala

```diff
@@ -0,0 +1,83 @@
+package org.locationtech.geomesa.convert.xml
+
+import java.io.ByteArrayInputStream
+import javax.xml.parsers.DocumentBuilderFactory
+import javax.xml.xpath.{XPathConstants, XPathExpression, XPathFactory}
+
+import com.google.common.base.Charsets
+import com.typesafe.config.Config
+import org.locationtech.geomesa.convert.Transformers.{EvaluationContext, Expr}
+import org.locationtech.geomesa.convert._
+import org.opengis.feature.simple.SimpleFeatureType
+import org.w3c.dom.NodeList
+
+import scala.collection.JavaConversions._
+
+class XMLConverter(val targetSFT: SimpleFeatureType,
+                   val idBuilder: Expr,
+                   val featurePath: Option[XPathExpression],
+                   val inputFields: IndexedSeq[Field]) extends ToSimpleFeatureConverter[String] {
+
+  private val docBuilder = {
+    val factory = DocumentBuilderFactory.newInstance()
+    factory.setNamespaceAware(false)
```

Comment:
it's possible we would need it to be able to parse some documents... I'll try to add one in. xsd's can be tricky though - a lot of them link to other xsds, which only works if they are actual files sitting on disk, and not loaded through the classpath (which we would probably need to do for deploying in storm). We might need to revisit this later if we can't get some real-world files to work.